packages/trpc/server/routers/viewer/bookings/_router.tsx

```diff
@@ -18,21 +20,26 @@ type BookingsRouterHandlerCache = {
 const UNSTABLE_HANDLER_CACHE: BookingsRouterHandlerCache = {};
 
 export const bookingsRouter = router({
-  get: authedProcedure.input(ZGetInputSchema).query(async ({ input, ctx }) => {
-    if (!UNSTABLE_HANDLER_CACHE.get) {
-      UNSTABLE_HANDLER_CACHE.get = await import("./get.handler").then((mod) => mod.getHandler);
-    }
+  get: authedProcedure
+    // Uncomment the below line to see a nasty error
+    // .meta({ openapi: { method: "GET", path: "/bookings" } })
```

Comment:
This is where I got stuck. I still don't know if it's because a GET endpoint expects a input in the form of an object, but uncommenting this makes the openAPI to explode.